Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM ruby:2.7
+
+WORKDIR /memfault/interrupt
+
+ENTRYPOINT ["./interrupt-cli"]
```

Comment:
nit: Maybe rename it to `./entrypoint.sh` to hint at that this is mostly intended to be run inside the Docker container?